### PR TITLE
Applying the Date and Time Location of the Format Class

### PR DESCRIPTION
**Pull request by leofreitas**
**Motivation and Context**
Applying the Date and Time Location of the Format Class

**How Has This Been Tested?**
Local and Travis

### CODE REVIEW

File: modules/Attendance/report_courseClassesNotRegistered_byDate.php [hunk 1]
```diff
@@ -194,7 +195,7 @@
                     echo $row['courseShort'].'.'.$row['class'];
                     echo '</td>';
                     echo '<td>';
-                    echo date('M j', $timestampStart).' - '. date('M j, Y', $timestampEnd);
+                    echo Format::dateReadable($dateStart, '%b %d').' - '. Format::dateReadable($dateEnd, '%b %d, %Y');
```
Comment (SKuipers) [inline modules/Attendance/report_courseClassesNotRegistered_byDate.php]: Check out the dateRangeReadable method here, it automatically handles whether the month & year should be shown for date ranges eg: Jan 5-15, 2019 vs Jan 5 - Feb 5, 2019, vs Dec 20 2018 - Jan 5, 2019

File: modules/Staff/staff_manage.php [hunk 2]
```diff
@@ -132,8 +132,8 @@
                 .'<br/><span style="font-size: 85%; font-style: italic">'.$person['initials']."</span>";
         });
 
-    $table->addColumn('type', __('Staff Type'))->width('20%')->translatable();
-    $table->addColumn('status', __('Status'))->width('10%')->translatable();
+    $table->addColumn('type', __('Staff Type'))->width('20%');
+    $table->addColumn('status', __('Status'))->width('10%');
```
Comment (SKuipers) [inline modules/Staff/staff_manage.php]: Are these no longer translatable? 🤔

Comment (leofreitas) [reply modules/Staff/staff_manage.php]: You're right! Yes, these are. 😬

File: modules/Attendance/report_courseClassesNotRegistered_byDate.php (hunk 1 above)
Comment (leofreitas) [reply modules/Attendance/report_courseClassesNotRegistered_byDate.php]: That's great!

**Comment by leofreitas** [comment]
I made the changes applying dataRangeReadable

**Comment by SKuipers** [comment]
Awesome thanks! Going to merge & test 👍

**Pull request merged**